service.autosubs/changelog.txt

```diff
@@ -0,0 +1,31 @@
+1.0.0 - schumi2004,CaTz, Mafarricos
```

Comment:
ahah please remove the file from the submission, it's still here

Reply:
Got clear goal now :) Removed changes.txt, removed (redundant) icon.png, left only three latest version descriptions in news